src/theme/palette.ts

```diff
@@ -14,6 +14,7 @@ export const createPalette = ({ dark }: { dark?: boolean }) => {
     primary: {
       main: mainGreen,
       contrastText: '#ffffff',
+      // TODO could not be needed anymore with change to primary green
       // default is lighten main color by 0.2
       light: lighten(mainGreen, 0.4),
```

Comment:
Look for usages of `primary.light` and consider a) replacing this with default lighten by 20% and/or b) replacing usage with `primary.main`

Reply:
@CarsonF  Given the new color is already a "brighter" green, it didn't really "stand out" much more when utilizing the `primary.light: lighten(mainGreen, 0.2)`, so I opted to refactor for the primary.main.  If that's ok, I'll push a commit with the changes, let me know.

Reply:
@atGit2021 Does it stand out more with the 40% as it is currently?
I'm just looking for what looks the best. Commit away.

Reply:
It didn't to me, but then again i'm not known for design :)